project-base/src/Shopsys/ShopBundle/Resources/config/services.yml

```diff
@@ -95,10 +95,6 @@ services:
         arguments:
```

Comment:
you removed the `ProductDataFixtureCsvReader` as well. Please check `services.yml` and `services_test.yml` for all deleted classes

Reply:
fixed (in one of the previous fixups :smile: )